source/include/ota_os_interface.h

```diff
@@ -213,7 +213,9 @@ typedef struct OtaTimerInterface
  */
 typedef struct OtaMallocInterface
 {
+    /* coverity[misra_c_2012_rule_21_3_violation] This is part of our interface . */
```

Comment:
Still too terse, but given the blanket nature of this deviation I recommend you put this in your misra.config instead.

Reply:
Updated comments in eb1c445.